src/ralph/domains/models/domains.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from django.db import models
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from dj.choices import Choices
+
+from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+from ralph.assets.models.assets import (
+    BusinessSegment,
+)
+
+from ralph.assets.models.assets import (
+    Service,
+)
+
+
+class DomainRegistrant(
+    PermByFieldMixin,
+    NamedMixin,
+    TimeStampMixin,
+    models.Model,
+):
+    pass
+
+
+class DomainStatus(Choices):
+    _ = Choices.Choice
+
+    active = _('Active')
+    pending_lapse = _('Pending lapse')
+    pending_transfer = _('Pending transfer away')
+    lapsed = _('Lapsed (inactive)')
+    transfered_away = _('Transfered away')
+
+
+class DomainContract(
```

Comment:
do we want to store only public domains? If not, then what about distinction between public and private domains? Or this would be stored in django_powerdns_dnssec and related?